documentRoot/.htaccess

```diff
@@ -0,0 +1,4 @@
+AuthType Basic
```

Comment:
Is it safe to reveal the contents of this file? I'm new to security, and authentication, and curious as to whether this file could be used against us.

Reply:
I don't think there's a security risk. All it's saying is that authentication is required which you'd learn from visiting the page anyways.